### ISSUE TITLE
[BUG] Blur does not work. 

### ISSUE THREAD

**Issue by Foxerbit** [Deletescape-Media/Lawnchair#1659]
Blur does not work, and makes pixels instead.

![IMG_20190717_045327_495](https://user-images.githubusercontent.com/52997187/61362210-cd320180-a84f-11e9-8584-1d5e563e1781.JPG)

## Description
I turned the blur on, it did not work and made orange, pink and yellow pixels. 
It's a bummer, because the blur is one of my favorite Lawnchair features.

## Expected Behavior
The blur should work as normal. 

## Actual Behavior
Did not work and made strange pixels. 

## Steps to Reproduce

1. All I did was turn on the blur.
NOTE: I have a normal Wallpaper, it's not live and does not change.
NOTE: Lawnchair didn't crash, it continued working normally, the blur just didn't work.

## Environment

* Device: Huawei Y5 Lite (2018)
* Android version: 8.1.0 (Android Go)
* Launcher version: Latest version as of July 17
* Rom:

## Logcat

 

```
your log here
```

**Comment by nyancrimew** [comment]
This is an issue with some specific devices, I am currently trying to get in possession of a device affected by this to debug it.

**Comment by macroboi** [comment]
Any updates on this yet?This to me seems like a huawei-specific issue my Honor 8A does that too

**Comment by geeqib23** [comment]
The same bug is there on my Realme C2 and Vivo Z1 Pro!

**Comment by fefelixa** [comment]
I've had the same on my Alcatel 3

**Comment by elyorch76** [comment]
On Essential Phone blur is not working, but also no strange pixels. Is not making anything at all, with normal wallpaper, not live, not changing

**Comment by DanGLVK** [comment]
@elyorch76 Are you using a live wallpaper? Because blur does not work on live wallpapers

**Comment by elyorch76** [comment]
No. Normal wallpaper

**Comment by DanGLVK** [comment]
> No. Normal wallpaper

Are you on the play store or prealpha builds?

**Comment by elyorch76** [comment]
> > No. Normal wallpaper
> 
> Are you on the play store or prealpha builds?

Play Store

**Comment by DanGLVK** [comment]
@elyorch76 Could you check on the prealpha builds? (You can find them on either telegram or apkmirror), they use a different blur algorithm than the play store builds so maybe it will work there

**Comment by tiagogaspar8** [comment]
Hey!

I have a OnePlus 8 and I'm having this issue too, on the stable, beta and alpha. One note, on my OnePlus 6 this wasn't an issue.

I know this is a bit of a sidetrack but are the stable builds ever getting updated? Or should I stick to the alpha series?

**Comment by DanGLVK** [comment]
> Hey!
> 
> I have a OnePlus 8 and I'm having this issue too, on the stable, beta and alpha. One note, on my OnePlus 6 this wasn't an issue.
> 
> I know this is a bit of a sidetrack but are the stable builds ever getting updated? Or should I stick to the alpha series?

Avoid setting wallpapers using the stock Oneplus wallpaper app as it is know to apply wallpapers in a way that breaks blur, about builds, things have been somewhat complicated internally and thus no one is having time to update the project for the time being (kinda sucks, but what can you do), so if you need stability then stick to the stable builds, atm there isn't a lot to gain on the prealphas if you are not rooted

**Comment by tiagogaspar8** [comment]
> Avoid setting wallpapers using the stock Oneplus wallpaper app as it is know to apply wallpapers in a way that breaks blur

WOW! FIXEDDDDD Thankssss!

> about builds, things have been somewhat complicated internally and thus no one is having time to update the project for the time being (kinda sucks, but what can you do), so if you need stability then stick to the stable builds, atm there isn't a lot to gain on the prealphas if you are not rooted

Shit, I'm sorry to hear that, if I can help in any way just say so! (I have no knowledge of coding but I can do some testing or help in any other way!)

**Comment by MacWrench** [comment]
I've been having the same issue after moving to a new phone. Had a Teracube 1 with Android 9 running the Play Store version of Lawnchair, blur worked great on that. I've now moved to the Teracube 2e with Android 10 and blur just gives me the rainbow dots. I'm using the same background photo (applied with google photos app), same version of Lawnchair, but no blur for me.

I tried the latest version on APKMirror (9.1 Alpha 3) as well; no dice
